src/test/java/dk/sdu/se_f22/sortingmodule/range/rangepublic/EqualsTest.java

```diff
@@ -0,0 +1,367 @@
+package dk.sdu.se_f22.sortingmodule.range.rangepublic;
+
+import dk.sdu.se_f22.sortingmodule.range.exceptions.IllegalMinMaxException;
+import org.junit.jupiter.api.*;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvFileSource;
+
+import java.time.Instant;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+
+public class EqualsTest {
+    @Nested
+    @DisplayName("Non-matching tests")
+    class NonMatchingTests {
+
+        @Nested
+        @DisplayName("Non-matching double filter tests")
+        class NonMatchingDoubleFilterTests {
+
+            @ParameterizedTest
+            @DisplayName("Double filter and ordinary object")
```

Comment:
The importance with test names is to communicate desired behaviour rather than testing methodology (that can be read in the test, with help from code comments)

Reply:
Done